blog.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
```

Comment:
Your HTML looks very good!
I like you use semantic tags, code formatting and working with GIT branches 👍